### PR TITLE
Reporting colors

### PR DESCRIPTION
**Pull request by ayanlee**
See #279  for context.

### CODE REVIEW

File: flask_monitoringdashboard/core/config/parser.py [hunk 1]
```diff
@@ -23,9 +23,14 @@ def parse_version(parser, header, version):
             # The file is specified by: 'ref: <location>'
             git_file = (open(os.path.join(git, 'HEAD')).read().rsplit(': ', 1)[1]).rstrip()
             # read the git-version
-            version = open(git + '/' + git_file).read()
-            # cut version to at most 6 chars
-            return version[:6]
+            version_file = os.path.join(git , git_file)
```
Comment (mircealungu) [inline flask_monitoringdashboard/core/config/parser.py]: it's not clear why this is part of the diff... it is not code written by you @ayanlee ... is it?

Comment (mircealungu) [reply flask_monitoringdashboard/core/config/parser.py]: ah. now i see. it's a PR to master. 
try to PR to development instead!

Comment (ayanlee) [reply flask_monitoringdashboard/core/config/parser.py]: My bad! I'll fix it quick.

**Comment by mircealungu** [comment]
oki, closing this and waiting for your PR to development @ayanlee

**Pull request closed without merging**